modules/monitoring-configuring-remote-write-storage.adoc

```diff
@@ -52,89 +60,14 @@ data:
   config.yaml: |
     prometheusK8s:
       remoteWrite:
-      - url: "https://remote-write.endpoint"
-        <endpoint_authentication_credentials>
-----
-+
-For `endpoint_authentication_credentials` substitute the credentials for the endpoint.
-Currently supported authentication methods are basic authentication (`basicAuth`) and client TLS (`tlsConfig`) authentication.
-+
-* The following example configures basic authentication:
-+
-[source,yaml]
-----
-basicAuth:
-  username:
-    <usernameSecret>
-  password:
-    <passwordSecret>
-----
-Substitute `<usernameSecret>` and `<passwordSecret>` accordingly. 
-+
-The following sample shows basic authentication configured with `remoteWriteAuth` for the `name` values and `user` and `password` for the `key` values. These values contain the endpoint authentication credentials:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
-data:
-  config.yaml: |
-    prometheusK8s:
-      remoteWrite:
-      - url: "https://remote-write.endpoint"
-        basicAuth:
-          username:
-            name: remoteWriteAuth
-            key: user
-          password:
-            name: remoteWriteAuth
-            key: password
-----
-+
-* The following example configures client TLS authentication:
-+
-[source,yaml]
-----
-tlsConfig:
-  ca:
-    <caSecret>
-  cert:
-    <certSecret>
-  keySecret:
-    <keySecret>
+      - url: "https://remote-write-endpoint.example.com" <1>
+        <endpoint_authentication_credentials> <2>
 ----
-Substitute `<caSecret>`, `<certSecret>`, and `<keySecret>` accordingly. 
 +
-The following sample shows a TLS authentication configuration using `selfsigned-mtls-bundle` for the `name` values and `ca.crt` for the `ca` `key` value, `client.crt` for the `cert` `key` value, and `client.key` for the `keySecret` `key` value:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: cluster-monitoring-config
-  namespace: openshift-monitoring
-data:
-  config.yaml: |
-    prometheusK8s:
-      remoteWrite:
-      - url: "https://remote-write.endpoint"
-        tlsConfig:
-          ca:
-            secret:
-              name: selfsigned-mtls-bundle
-              key: ca.crt
-          cert:
-            secret:
-              name: selfsigned-mtls-bundle
-              key: client.crt
-          keySecret:
-            name: selfsigned-mtls-bundle
-            key: client.key
-----
+<1> The URL of the remote write endpoint.
+<2> The authentication method and credentials for the endpoint.
+Currently supported authentication methods are AWS Signature Version 4, bearer token, basic authentication, OAuth 2.0, and TLS client. 
+See _Supported remote write authentication settings_ below for sample configurations of supported authentication methods.
 
 . Add write relabel configuration values after the authentication credentials:
```

Comment:
I think this setting is optional for configure remote write storage, but keep it is fine